ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -315,6 +318,32 @@ - (void)errorViewDidSelectRetry:(EXErrorView *)errorView
 
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations
 {
+  NSString *validatedVersion = [[EXVersions sharedInstance] availableSdkVersionForManifest:_appRecord.appLoader.manifest];
+  if ([@[@"33.0.0", @"34.0.0", @"35.0.0"] containsObject:validatedVersion]) {
```

Comment:
this still doesn't look well 😛 I guess we can write a method in EXVersions that can help us in such cases (it has additional `versionComponents` method in `NSString` that you could use)